source/Calamari.Tests/Helpers/TestEnvironmentExtended.cs

```diff
@@ -0,0 +1,35 @@
+using Assent;
+using Assent.Namers;
+using Calamari.Testing.Helpers;
+
+namespace Calamari.Tests.Helpers
+{
+    public static class TestEnvironmentExtended
+    {
+        static bool IsCi => TestEnvironment.IsCI;
```

Comment:
This can be `private`, but I think it is better to just remove it and use `TestEnvironment.IsCI` directly.

Reply:
Good point, fixed.